build/rpm/pmm2-client.spec

```diff
@@ -81,6 +81,7 @@ install -m 0660 example.prom $RPM_BUILD_ROOT/usr/local/percona/pmm2/collectors/t
 install -m 0660 queries-mysqld.yml $RPM_BUILD_ROOT/usr/local/percona/pmm2/collectors/custom-queries/mysql/low-resolution/
 install -m 0660 queries-mysqld.yml $RPM_BUILD_ROOT/usr/local/percona/pmm2/collectors/custom-queries/mysql/medium-resolution/
 install -m 0660 queries-mysqld.yml $RPM_BUILD_ROOT/usr/local/percona/pmm2/collectors/custom-queries/mysql/high-resolution/
+install -m 0660 queries-mysqld-group-replication.yml $RPM_BUILD_ROOT/usr/local/percona/pmm2/collectors/custom-queries/mysql/high-resolution/
```

Comment:
why only for high-resolution?

Reply:
There is no reason to collect the same metrics with different resolutions.

Reply:
why do we do this for other metrics?

Reply:
It would lead to situation when we have bunch of exactly the same metrics from different metric resolution collectors.
e.g.
interval: 1m
hr: 60 metrics
mr: 10 metrics
lr: 1 metric

So we have 
12 - concurrent metrics for equal timestamps.

It's definitely break used formulas in dashboards.